apps/bixarena/api/src/main/resources/db/migration/V1.0.0__create_tables.sql

```diff
@@ -29,6 +29,21 @@ CREATE TABLE api.model (
 CREATE INDEX idx_api_model_license ON api.model(license);
 CREATE INDEX idx_api_model_active ON api.model(active);
 
+-- Model errors table
+CREATE TABLE api.model_error (
+  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  model_id UUID NOT NULL REFERENCES api.model(id) ON DELETE CASCADE,
+  code INTEGER,
+  message VARCHAR(1000) NOT NULL,
+  battle_id UUID,
+  round_id UUID,
```

Comment:
We should link these IDs to their corresponding properties. This will ensure that the battle with the ID specified and round with the ID specified exist in the database before we insert this error.